### PR TITLE
Quilt 3

### PR DESCRIPTION
**Pull request by kevinemoore**
## Description
Initial commit of Quilt 3 based on the `t4` API and catalog (https://github.com/quiltdata/t4).

## TODO

Use this section for work-in-progress pull requests. If you're using this section,
please tag your PR "WIP". 

- [x] Fix and merge v2 & v3 documentation
- [x] Update CI
- [x] Test catalog v3

### CODE REVIEW

File: api/python/quilt3/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""T4 API"""
```
Comment (stevededalus) [inline api/python/quilt3/__init__.py]: T4 API?

File: api/python/quilt3/admin.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+"""Provides admin-only functions for T4."""
```
Comment (stevededalus) [inline api/python/quilt3/admin.py]: s/T4/Quilt3

File: README.md [hunk 3]
```diff
@@ -3,126 +3,37 @@
...
-Alternatively, users can run their own registries (refer to the
-[registry documentation](https://github.com/quiltdata/quilt/blob/master/registry/README.md)).
-
-## Architecture
-Quilt consists of three components. See the
-[contributing docs](https://github.com/quiltdata/quilt/blob/master/docs/CONTRIBUTING.md)
-for further details.
-    
-<img width="640" src="https://raw.githubusercontent.com/quiltdata/resources/master/img/arch.png" />
+* [Roadmap](https://github.com/quiltdata/t4/blob/master/Roadmap.md)
```
Comment (stevededalus) [inline README.md]: url points to old repo

File: api/python/setup.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        tag = os.getenv('CIRCLE_TAG')
+
+        if tag != VERSION:
+            info = "Git tag: {0} does not match the version of this app: {1}".format(
+                tag, VERSION
+            )
+            sys.exit(info)
+
+setup(
+    name="quilt3",
```
Comment (stevededalus) [inline api/python/setup.py]: Does this work? I don't see a `quilt3` package in pypi

File: api/python/quilt3/admin.py (hunk 2 above)
Comment (stevededalus) [reply api/python/quilt3/admin.py]: Looks like most of the top level doc strings have this issue

**Comment by ResidentMario** [comment]
I've created a [`quilt-2-master`](https://github.com/quiltdata/quilt/tree/quilt-2-master) branch which will continue to serve as the `master` branch for the Quilt 2 product.

In this PR docs for Quilt 2 are under a `v2` directory and docs for Quilt 3 are under a `v3` directory. That's not really how Gitbooks is intended to work, and in the current organization I believe the `v2` files will simply be inaccessible to end users. What we want to do is add the `quilt-2-master` branch to the documentation version selector, which can be done via the Gitbooks UI&mdash;*after* this PR has merged.

I recommend undoing the docs-related changes. After merging this PR, I can make the necessary changes to Gitbook configuration.

**Pull request merged**